src/components/YourCompany/CorrectBusinessType.vue

```diff
@@ -65,61 +67,65 @@
+              <span v-if="!helpToggle">Learn More</span>
+              <span v-else>Hide Learn More</span>
+            </div>
+
+            <!-- Help Section -->
+            <section v-show="helpToggle" class="mx-8 my-7">
+              <p class="subtitle mb-2">Benefit Provision</p>
+              <p class="info-text">
+                A benefit company must include a benefit provision (a statement by the company of its public benefits
+                and its commitments to promote those public benefits and to conduct business in a responsible and
+                sustainable manner) in the company's articles. The benefit provision can be added as part of the
+                company's existing articles or as part of new articles.
+              </p>
+              <div class="provision-help">
+                <p class="subtitle">Part 1 - Benefit Provision</p>
+                <ol class="info-text ml-2 pl-0">
+                  <li style="list-style:none;">
+                    <ol>
+                      <li class="mb-4">
+                        <span class="ml-4">The Company commits to promote the following public benefits:</span><br>
+                        <span class="ml-2">[List your public benefits in this section]</span>
+                      </li>
+                      <li>
+                        <span class="ml-4">The Company commits</span><br>
+                        <div class="ml-5">
                       <span class="ml-n3">i) to conduct the benefit company's business in a responsible and sustainable
                       manner;</span><br>
-                      <span class="ml-n3">ii) to promote the public benefits specific in paragraph 1.1</span>
-                    </div>
+                          <span class="ml-n3">ii) to promote the public benefits specific in paragraph 1.1</span>
+                        </div>
+                      </li>
+                    </ol>
                   </li>
                 </ol>
-              </li>
-            </ol>
-          </div>
-        </section>
-
-        <!-- Confirm Articles Checkbox -->
-        <div class="pt-2 pr-2">
-          <v-checkbox
-            v-model="confirmArticles"
-            id="confirm-articles-checkbox"
-            :label="`The company has completed a set Benefit Company Articles containing a benefit provision, and a copy
-             of these articles has been added to company's record book.`"
-          ></v-checkbox>
-        </div>
+              </div>
+            </section>
+
+            <!-- Confirm Articles Checkbox -->
+            <div class="pr-2">
+              <v-checkbox
+                v-model="confirmArticles"
+                id="confirm-articles-checkbox"
+                :label="`The company has completed a set Benefit Company Articles containing a benefit provision, and a
+              copy of these articles has been added to company's record book.`"
```

Comment:
You might to extract this to a property or getter, to avoid fancy quotes and multiple lines. (Multiple lines are affected by the CSS `white-space` property, and it may be hard to tell what that property value is here.)

Reply:
Will do!